app/lib/discourse_automation/scripts/auto_responder.rb

```diff
@@ -22,6 +22,11 @@
 
     answers = Set.new
     tuples = JSON.load(fields.dig('word_answer_list', 'value'))
+
+    unless tuples.present?
+      next
+    end
```

Comment:
This could be a positive one-liner 😉 

```suggestion
    next if tuples.blank?
```